src/main/java/org/mifos/identityaccountmapper/service/AccountLookupReadService.java

```diff
@@ -37,7 +36,7 @@ public AccountLookupReadService(MasterRepository masterRepository, ErrorTracking
         this.objectMapper = objectMapper;
     }
 
-    @Cacheable(value = "accountLookupCache", key = "#payeeIdentity")
+    // @Cacheable(value = "accountLookupCache", key = "#payeeIdentity")
```

Comment:
can we enable/ disable this with a configuration than commenting the lines

Reply:
Uncommented these lines by comment enable config annotations